Meadowlark-js/jest-mongodb-config.js

```diff
@@ -1,11 +1,11 @@
 module.exports = {
   mongodbMemoryServerOptions: {
     binary: {
-      version: '4.0.28',
+      version: '4.4.0',
```

Comment:
This bothered me a bit at first, because testing against a newer version doesn't prove that we're only using 4.0-compatible code. However, so long as our e2e tests are using Mongo 4.0, then we're in good shape.